plugins/cros_ec-usb/fu-plugin-cros_ec-usb.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2020 Benson Leung <bleung@chromium.org>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-plugin-vfuncs.h"
+#include "fu-hash.h"
+
+#include "fu-cros_ec-usb-device.h"
+
+void
+fu_plugin_init (FuPlugin *plugin)
+{
+	fu_plugin_set_build_hash (plugin, FU_BUILD_HASH);
+	fu_plugin_set_device_gtype (plugin, FU_TYPE_CROS_EC_USB_DEVICE);
+	g_print ("fu-plugin-cros_ec_usb fu_plugin_init!\n");
```

Comment:
I suspect this was a debugging line originally, but it should be unnecessary now